libs/api/domains/education/src/lib/models/competence.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType('EducationCompulsorySchoolCourseCompetence')
+export class CourseCompetence {
+  @Field() //B+
+  competencyGrade!: string
+
+  @Field({ nullable: true }) // A1
+  competenceStatus?: string
```

Comment:
Is it possible to expound on these comments (or remove them)?

Reply:
Should be removed